caddytest/integration/caddyfile_adapt/php_fastcgi_index_off.txt

```diff
@@ -7,6 +7,9 @@ php_fastcgi localhost:9000 {
     env VAR2 value2
     root /var/www
     index off
+	dial_timeout 3s
+	read_timeout 10s
+	write_timeout 20s
```

Comment:
For some reason, all other indents are spaces, not tabs, so these look weird. Can we normalize this file to use tabs?

Reply:
Oh, oops, ok